website/pages/partials/builder/scaleway/Config-required.mdx

```diff
@@ -1,15 +1,19 @@
 <!-- Code generated from the comments of the Config struct in builder/scaleway/config.go; DO NOT EDIT MANUALLY -->
 
 - `access_key` (string) - The AccessKey corresponding to the secret key.
+  Will be fetched first from the [scaleway configuration file](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md).
   It can also be specified via the environment variable SCW_ACCESS_KEY.
 
 - `secret_key` (string) - The SecretKey to authenticate against the Scaleway API.
+  Will be fetched first from the [scaleway configuration file](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md).
```

Comment:
This documentation changed from the go struct here https://github.com/hashicorp/packer/blob/70cb4097e17efa69d270c476eaf23e03ff443983/builder/scaleway/config.go#L30-L32

So that the go code is our source of truth 🙂 .